pkgs/applications/misc/houdini/runtime.nix

```diff
@@ -29,11 +29,11 @@ let
   license_dir = "~/.config/houdini";
 in
 stdenv.mkDerivation rec {
-  version = "17.0.352";
+  version = "17.5.327";
   pname = "houdini-runtime";
   src = requireFile rec {
     name = "houdini-${version}-linux_x86_64_gcc6.3.tar.gz";
```

Comment:
Still called `...._gcc6.3.tar.gz` with the bump to 17.5 ?

Reply:
Yes, still gcc6.3